src/main/java/github/scarsz/discordsrv/objects/managers/JdbcAccountLinkManager.java

```diff
@@ -190,6 +209,41 @@ public JdbcAccountLinkManager() throws SQLException {
                 }
             }
         }
+
+        dropExpiredCodes();
+
+        final ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat("DiscordSRV - Key Expiry Scheduler").build();
+        keyExpiryScheduler = Executors.newSingleThreadScheduledExecutor(threadFactory);
+
+        final String jedisHost = DiscordSRV.config().getString("Experiment_RedisHost");
+        if (jedisHost == null || jedisHost.equals("HOST")) {
+            jedisPool = null;
+        } else {
+            final int jedisPort = DiscordSRV.config().getInt("Experiment_RedisPort");
+            final String jedisPassword = DiscordSRV.config().getString("Experiment_RedisPassword");
+            if (jedisPassword == null || jedisPassword.equals("")) {
+                jedisPool = new JedisPool(new JedisPoolConfig(), jedisHost, jedisPort);
+            } else {
+                jedisPool = new JedisPool(new JedisPoolConfig(), jedisHost, jedisPort, 0, jedisPassword);
+            }
+        }
+    }
+
+    private void dropCode(UUID uuid, String code) {
+        final String uuidString = uuid.toString();
+        try (final PreparedStatement statement = connection.prepareStatement("delete from " + codesTable + " where `uuid` = ? and `code` = ?")) {
+            statement.setString(1, uuidString);
+            statement.setString(2, code);
+            statement.executeUpdate();
+            if (jedisPool != null) {
+                try (Jedis jedis = jedisPool.getResource()) {
+                    jedis.del("linking/code/" + code);
+                    jedis.del("linking/uuid/" + uuidString);
```

Comment:
`Jedis#del`'s accept multiple inputs, eg.
```suggestion
                    jedis.del("linking/code/" + code, "linking/uuid/" + uuidString);
```